projects/igniteui-angular/src/lib/query-builder/query-builder-tree.component.ts

```diff
@@ -5,7 +5,7 @@ import {
     Output,
     TemplateRef
 } from '@angular/core';
-import { getLocaleFirstDayOfWeek, NgTemplateOutlet, NgClass, DatePipe } from '@angular/common';
+import { NgTemplateOutlet, NgClass, DatePipe } from '@angular/common';
```

Comment:
Do we plan to change the DatePipe used in the templates? Currently it's using angular's date pipe, so it's not using our `IgxDateFormatterPipe` with the Intl implementation.